igcollect/redis.py

```diff
@@ -41,16 +44,19 @@ def main():
         print(template.format(metric, redis_stats[metric]))
 
 
-def redis_pwd():
-    """Get the Redis password from the configuration"""
+def get_redis_conf(*args):
+    """Get requested parameters from the configuration"""
     with open("/etc/redis/redis.conf") as fd:
-        secret_cfg = fd.read().splitlines()
+        content = fd.read().splitlines()
 
-    for line in secret_cfg:
-        line = line.strip()
-        if line.startswith("requirepass"):
-            return line.split(" ")[1].strip()
-    return ''
+    cfg = {}
+    for line in content:
+        parts = line.strip().split(' ')
```

Comment:
Is it really correct with `split(' ')` rather than just `split()`. `split()` already split on all whitespaces, `split(' ')` just splits on one whitespace. Maybe it is really correct like that, but wondering if it is really meant to be or just accidentally called like that.

Reply:
With `split()` the former `strip()` might also be not needed anymore.